app/uk/gov/hmrc/apinotificationpull/controllers/NotificationsController.scala

```diff
@@ -45,9 +49,21 @@ class NotificationsController @Inject()(apiNotificationQueueService: ApiNotifica
   }
 
   def delete(notificationId: String): Action[AnyContent] =
-    (headerValidator.validateAcceptHeader andThen headerValidator.validateXClientIdHeader).async {
-      Future.successful(NotFound)
+    (headerValidator.validateAcceptHeader andThen headerValidator.validateXClientIdHeader).async { implicit request =>
+
+    apiNotificationQueueConnector.getById(notificationId)
```

Comment:
You could call the service here.
And the service calls the connector.
Do you agree?